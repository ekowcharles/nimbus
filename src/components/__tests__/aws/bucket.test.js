import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Bucket from '../../aws/bucket'

describe('Bucket', () => {
  it('renders databases', () => {
    const wrapper = shallow(
      <Bucket
        bucket={
          {
            "name": "nimbusmain",
            "region": "us-east-1"
          }
        }
        settings={
          {
            "hide": false,
            "hideName": false,
            "hideStatus": false,
          }
        }
      />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})