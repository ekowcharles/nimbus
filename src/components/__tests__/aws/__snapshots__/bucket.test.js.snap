// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bucket renders databases 1`] = `
<Resource
  hideName={false}
  hideStatus={false}
  image="bucket.png"
  name="nimbusmain"
  updateResourceAttributes={[Function]}
/>
`;
